scripts/rest_client.py

```diff
@@ -237,8 +244,9 @@ def print_commands():
 
 def main(args):
 
-    if args.get("help"):
+    if args.get("show"):
```

Comment:
Why 'show'? 'help' was okay to print commands usage...

Reply:
but "help" did never work. 
there's 2 types of help for this - "--help" - which shows available command line switches - and "--show" - showing available commands.

The correct argument here was always show ... --help is not populated but exits directly from argparse, so never reaches this line.

Reply:
the name of the `show` command is misleading. Okay for now, but a better way to output it should be 

the list of the commands can be shown (there are many alternatives):
* in the `usage` attribute of the argparse parser (will be printed before arguments description in the output of `--help`)
* as output of `help` command (so `--help` prints options and `help` command prints commands)
* as output of `show commands`, `show_commands`, `help commands`, `--help commands`
* etc.

Using `show` for this is misleading.

2. Pythonic `:return:`s in the list of commands is ugly.

Just imho

Reply:
> Pythonic :return:s in the list of commands are ugly.

Basically, we're printing the docstrings - which removes the need to document each command seperately (which will lead to mistakes if it's pulled into one string).

I'm open to suggestions - but while being ugly - it works and documents stuff (and honestly, i don't think the rest-client is that heavily used) - who can, will probably use telegram ...

> as output of show commands, show_commands, help commands, --help commands

one way to get to this list is indeed `python scripts/rest_client.py help` (actually ANY command which does not exist) ... but requires the config to be found now ...

Reply:
pls review my commit

Reply:
overall, the output did improve by removing the :return: lines.

However,  the "method" of doing this also removed some information about optional (or mandatory) parameters.

Please have a look at my latest change to this.

*side note to self*: we should update the documentation regarding this with the outcome of these changes ...